pkg/ecdsa/tss/protocol_recovery.go

```diff
@@ -0,0 +1,147 @@
+package tss
+
+import (
+	"context"
+	cecdsa "crypto/ecdsa"
+	"fmt"
+	"time"
+
+	"github.com/keep-network/keep-core/pkg/net"
+)
+
+// BroadcastRecoveryAddress broadcasts and receives the BTC recovery addresses
+// of each client so that each client can retrieve the underlying bitcoin in
+// the case that a keep is terminated.
+func broadcastRecoveryAddress(
```

Comment:
You can just make this exported in-place here and drop the exported passthrough stub in tss.go, fwiw. The package's exported functions are pulled from all the files in the package.